geonode_mapstore_client/context_processors.py

```diff
@@ -26,6 +26,7 @@ def resource_urls(request):
         'DEFAULT_MAP_CRS': getattr(settings, "DEFAULT_MAP_CRS", 'EPSG:3857'),
         'DEFAULT_MAP_ZOOM': getattr(settings, "DEFAULT_MAP_ZOOM", 0),
         'DEFAULT_TILE_SIZE': getattr(settings, "DEFAULT_TILE_SIZE", 512),
+        'DEFAULT_MAX_UPLOAD_SIZE': getattr(settings, "DEFAULT_MAX_UPLOAD_SIZE", 104857600),
```

Comment:
@marthamareal @giohappy 
Should we gat the upload size from the db instead? and should we have two different variables?

Reply:
updated